config/dev-disco.js

```diff
@@ -25,4 +25,13 @@ module.exports = {
 
   // https://sentry.prod.mozaws.net/operations/addons-frontend-disco-dev/
   publicSentryDsn: 'https://560fc81d9fd14266b99bda032de23c52@sentry.prod.mozaws.net/184',
+
+  enableFeatureDiscoTaar: true,
+  discoParamsToUse: [
```

Comment:
Where does this come from? And why is it in the config?

Reply:
I am overriding what's in `default-disco`. I was removing clientId but it's moot now so you said I should keep this as the same name. So I'll revert this: )